src/github.com/getlantern/errors/errors.go

```diff
@@ -93,6 +93,18 @@ type Error interface {
 	error
 	context.Contextual
 
+	// ErrorClean returns a non-parameterized version of the error whenever
+	// possible. For example, if the error text is:
+	//
+	//     unable to dial www.google.com caused by: i/o timeout
+	//
+	// ErrorClean might return:
+	//
+	//     unable to dial %v caused by: %v
+	//
+	// This can be useful when performing analytics on the error.
+	ErrorClean() string
```

Comment:
I snuck this in as a bonus.  This makes it easier to aggregate errors in Loggly and Borda.